po/id.po

```diff
@@ -0,0 +1,18458 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Comprehensive Rust 🦀\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2023-06-04 21:58+0700\n"
+"Last-Translator: Aji Prio Sejati <ajiprio@hotmail.com>\n"
+"Language-Team: Indonesian\n"
+"Language: id\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#: src/SUMMARY.md:3
+msgid "Welcome to Comprehensive Rust 🦀"
+msgstr "Selamat datang di Rust yang Komprehensif🦀"
```

Comment:
Thank you so much! I'll definitely go through the guide. Since it's my first time contributing to the community, please feel free to let me know if I make any mistakes. I appreciate your support!

Reply:
Thanks and thank you for helping out!

Reply:
Please resolve this conversation when you've update the course title.

Also, please take a look at reverting the changes to the unrelated files.

Thanks!